pkg/minikube/constants/constants.go

```diff
@@ -55,6 +55,8 @@ const (
 	DefaultMemory   = 1024
 	DefaultCPUS     = 1
 	DefaultVMDriver = "virtualbox"
+
+	DefaultKubernetesVersion = ""
```

Comment:
Instead of this, you could get the actual value of kubernetes from k8s.io/kubernetes/pkg/version.GetVersion().

We set that automatically during a build, it'll look something like v0.1.3